src/middlewares.js

```diff
@@ -23,6 +23,41 @@ const getMountForRequest = function (req) {
   return req.protocol + '://' + req.get('host') + mountPath;
 };
 
+const checkIPRange = (ip, ranges = []) => {
```

Comment:
Could you add more tests to cover all the cases we have identified in the issue? Either by extending the tests you've added and running them in a loop for a array of ranges to test, or by testing this `checkIPRange` directly and making sure that it can handle all these ranges.

Reply:
From the issue:

> Adding ::/0 should work but doesn't.

Fixed, added test

> Adding :: should work but doesn't.

Is `::` a valid ipv6 address?

> Not sure if adding ::1 should work as well, it's the IPv6 notation for "localhost", but we haven no evidence that ::ffff:192.168.0.156 is in fact localhost.

Added test for `::1` as localhost

Reply:
`::` is an alternative notation for `0000:0000:0000:0000:0000:0000:0000:0000`. It's an unspecified IPv6 address which acts as a placeholder a host can assume before it has been assigned an IPv6 address.

There may not be much use for supporting this if we consider usage only within the specs. However it's a valid IPv6 address and just as with `0.0.0.0` there may be OS specific (non RFC conform) interpretations, so we should make sure to handle it in some way and document that.